docs/resources/logs_index.md

```diff
@@ -3,17 +3,18 @@
 page_title: "datadog_logs_index Resource - terraform-provider-datadog"
 subcategory: ""
 description: |-
-  Provides a Datadog Logs Index API resource. This can be used to create and manage Datadog logs indexes.
+  Provides a Datadog Logs Index API resource. This can be used to create and manage Datadog logs indexes.Note: It is not possible to delete logs indexes through Terraform, so an index remains in your account after the resource is removed from your terraform config. Reach out to support to delete a logs index.
```

Comment:
```suggestion
  Provides a Datadog Logs Index API resource. This can be used to create and manage Datadog logs indexes. Note: It is not possible to delete logs indexes through Terraform, so an index remains in your account after the resource is removed from your terraform config. Reach out to support to delete a logs index.
```

Reply:
Thanks for the review !
Since this file is generated, I can't really change it, but this part isn't rendered anyway on the terraform docs website, so it's fine :)

Reply:
👍